.travis.yml

```diff
@@ -93,8 +72,20 @@ before_install:
   - chmod +x ./docs/scripts/sphinx_build_link_check.sh
   - chmod +x ./docs/scripts/doc8_style_check.sh
 
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    curl -LO https://raw.githubusercontent.com/GiovanniBussi/macports-ci/master/macports-ci;
```

Comment:
What is this?

Reply:
I already told yoy about that error in travis ci . Thhere is bug that .congure run on py2 instead of py3 .

Reply:
ok, got it: 
1. can you install it from here instead? https://github.com/nexB/macports-ci
2. can you explain with a # comment why you are doing there in the file?

Reply:
Your repo is forked from that sources from where i have installed.

Reply:
hey, now i m using nexb macport ci repo.